e2e/src/step-definitions/hooks.ts

```diff
@@ -1,13 +1,24 @@
-import { After, Status } from 'cucumber';
-import { browser } from 'protractor';
+import {After, Before, Status} from 'cucumber';
+import {browser} from 'protractor';
 import * as fs from 'fs';
 
-After(async function(scenario) {
+Before({tags: '@MediaViewer'}, async () => {
+  await browser.waitForAngularEnabled(false);
+  return await browser.driver.navigate().to(browser.baseUrl);
```

Comment:
what's this return statement for?

Reply:
that's return driver object... let's discuss this...